.bazelci/presubmit.yml

```diff
@@ -1,26 +1,60 @@
 ---
-platforms:
-  ubuntu2204:
-    build_targets:
-    - "//..."
+tasks:
+  web_testing_ubuntu2004:
+    name: "rules_webtesting test in ubuntu"
+    platform: ubuntu2004
     test_flags:
-    - "--test_tag_filters=-noci,-external,-native,-sauce"
+    - "--test_tag_filters=-noci,-sauce"
     test_targets:
     - "//..."
-  macos:
-    build_targets:
-    - "//..."
+    - "-//go/webdriver:go_default_test_chromium-local" # Missing shared libs
+    - "-//go/webdriver:go_default_test_firefox-local" # Missing shared libs
+    - "-//go/webtest:go_default_test_chromium-local" # Missing shared libs
+    - "-//go/webtest:go_default_test_firefox-local" # Missing shared libs
+    - "-//go/wtl/proxy/driverhub/quithandler:go_default_test_chromium-local" # Missing shared libs
+    - "-//go/wtl/proxy/driverhub/quithandler:go_default_test_firefox-local" # Missing shared libs
+    - "-//go/wtl/proxy/driverhub/scripttimeout:go_default_test_chromium-local" # Missing Missing shared libs
+    - "-//go/wtl/proxy/driverhub/scripttimeout:go_default_test_firefox-local" # Missing shared libs
+    - "-//javatests/com/google/testing/web:WebTestTest_chromium-local" # Missing shared libs
+    - "-//javatests/com/google/testing/web:WebTestTest_firefox-local" # Missing shared libs
+    - "-//scalatests/com/google/testing/web:WebTestTest_chromium-local" # Missing shared libs
+    - "-//scalatests/com/google/testing/web:WebTestTest_firefox-local" # Missing shared libs
+    - "-//testing/web:webtest_test_chromium-local" # Missing shared libs
+    - "-//testing/web:webtest_test_firefox-local" # Missing shared libs
+  web_testing_macos:
+    name: "rules_webtesting test in ubuntu"
+    platform: macos
     test_flags:
-    - "--test_tag_filters=-noci,-sauce,-external,-native"
+    - "--test_tag_filters=-noci,-sauce"
```

Comment:
So the only test the would benefit from this is `//go/wtl/proxy:go_default_test`, which seems like marginal benefit considering that we don't have a concrete plan on how to get chrome/firefox test otherwise working on mac (with bazel CI at least).

I would maybe lean toward totally disabling `chromium-local` and `firefox-local` for mac.

For linux it's fine to list out the disabled targets since we know how to fix them (and will in a follow up)